doc/help/CMakeLists.txt

```diff
@@ -16,5 +16,11 @@ if (RONN_LOC)
 		generate_manpage (${NAME} SECTION 7)
 	endforeach ()
 else ()
-	message (WARNING "Sorry, I cannot find ronn, thus I cannot build the man pages. Please install https://github.com/rtomayko/ronn.")
+	message ("Sorry, I cannot find ronn, thus I will install man pages directly from the repository.")
+	set (MAN_PAGES_DIR "${CMAKE_SOURCE_DIR}/doc/man")
+
+	if (INSTALL_DOCUMENTATION)
```

Comment:
What about providing a target that rebuilds all man pages using update-manpages? (Maybe EXCLUDE_FROM_ALL).

Reply:
I do not think that makes that makes much sense, since calling `ninja update-manpages` or `update-manpages` does not make that much of a difference in my opinion.